src/system/pokemon-data.ts

```diff
@@ -148,7 +148,7 @@ export default class PokemonData {
         .map((m: any) => new PokemonMove(m.moveId, m.ppUsed, m.ppUp, m.virtual, m.maxPpOverride));
       if (!forHistory) {
         this.status = source.status
-          ? new Status(source.status.effect, source.status.toxicTurnCount, source.status.sleepTurnsRemaining)
+          ? new Status(source.getStatusEffect(), source.status.toxicTurnCount, source.status.sleepTurnsRemaining)
```

Comment:
```suggestion
          ? new Status(source.getStatusEffect(true), source.status.toxicTurnCount, source.status.sleepTurnsRemaining)
```
Hmm, I wonder if this one should be changed at all...